server/controllers/admin.py

```diff
@@ -557,6 +557,10 @@ def view_scores(cid, aid):
 
     if not include_all:
         query = query.filter_by(archived=False)
+
+    # sort scores by submission time in descending order, to match front end display
+    query = query.order_by(Score.created.desc())
```

Comment:
Can you just chain .order_by to the original query on line `555` since this it not conditionally happening.